### PR TITLE
Create 00_tokenomics.subpage.md

### PR DESCRIPTION
**Pull request by bjoernek**
add new page on tokenomics to how-it-works

### CODE REVIEW

File: how-it-works/5_governance/00_tokenomics.subpage.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+shareImage: /img/how-it-works/direct-integration-with-bitcoin.600x300.jpg
+slug: tokenomics
+---
+
+# Tokenomics of the Internet Computer
+
+
+### The ICP utility token
+
+The Internet Computer (IC) makes use of a utility token called ICP. This token is used for the following functions on the platform: First, any ICP holder can participate in the governance of the IC by staking ICP in order to vote on proposals and earn voting rewards. Second you can transform ICP into cycles, which are used as fuel for computation. Third, ICP tokens are used to pay the entities who provide compute capacity by operating node machines. Last but not least, ICP can be used in order to invest in token sales of decentralized autonomous organizations (DAOs) on the IC. We will elaborate on these four use cases in the following. In addition to the aforementioned platform use cases, ICP can also be used as a medium of exchange, i.e., to pay for goods and services like NFTs, subscriptions, etc.
```
Comment (Dfinity-Bjoern) [inline how-it-works/5_governance/00_tokenomics.subpage.md:13]:
```suggestion
The Internet Computer (IC) makes use of a utility token called ICP. This token is used for the following functions on the platform: First, any ICP holder can participate in the governance of the IC by staking ICP in order to vote on proposals and earn voting rewards. Second, you can transform ICP into cycles, which are used as fuel for computation. Third, ICP tokens are used to pay the entities who provide compute capacity by operating node machines. Last but not least, ICP can be used in order to invest in token sales of decentralized autonomous organizations (DAOs) on the IC. We will elaborate on these four use cases in the following. In addition to the aforementioned platform use cases, ICP can also be used as a medium of exchange, i.e., to pay for goods and services like NFTs, subscriptions, etc.
```

File: how-it-works/5_governance/00_tokenomics.subpage.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Since, the daily reward amount is independent of the overall amount of staked ICP in the system, participants receive larger rewards if overall staking and participation in governance decreases. This mechanism creates a natural incentive to stake ICP and participate in governance. As of November 2022, 266M ICP is staked, corresponding to 54% of the total supply. A significant part of staked ICP, namely 123M ICP (i.e., 46%), is staked for the maximum time of 8 years expressing the long-term commitment of these stakers.
+
+Neurons can be configured to vote automatically by following the votes of other neurons, in an advanced form of “liquid democracy.” Neurons that vote automatically still receive their full share of the voting reward, as they enable the IC community to reach decisions securely and quickly.
+
+The following graph depicts annualized voting rewards as a function of the staking time as of November 1, 2022. For current estimates of annualized voting rewards, refer to the IC Dashboard’s [Governance page](https://dashboard.internetcomputer.org/governance).
+
+![](/img/how-it-works/voting_rewards.png)
```
Comment (sesi200) [inline how-it-works/5_governance/00_tokenomics.subpage.md:27]: image does not exist

File: how-it-works/5_governance/00_tokenomics.subpage.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+For participation in governance, neurons receive voting rewards which can be converted into newly minted ICP. Every day the IC calculates a voting reward pot according to a schedule, which it then divides among all eligible neurons according to their relative voting power. The schedule for voting rewards is designed to incentivize early adoption: Initially at genesis, 10% of the total supply of ICP is distributed in voting rewards on an annualized basis. Over the course of eight years, this number falls to 5%.
```
Comment (Dfinity-Bjoern) [inline how-it-works/5_governance/00_tokenomics.subpage.md:19]:
```suggestion
For participation in governance, neurons receive voting rewards which can be converted into newly minted ICP. Every day, the IC calculates a voting reward pot according to a schedule, which it then divides among all eligible neurons according to their relative voting power. The schedule for voting rewards is designed to incentivize early adoption: Initially at genesis, 10% of the total supply of ICP is distributed in voting rewards on an annualized basis. Over the course of eight years, this number falls to 5%.
```

File: how-it-works/5_governance/00_tokenomics.subpage.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Since, the daily reward amount is independent of the overall amount of staked ICP in the system, participants receive larger rewards if overall staking and participation in governance decreases. This mechanism creates a natural incentive to stake ICP and participate in governance. As of November 2022, 266M ICP is staked, corresponding to 54% of the total supply. A significant part of staked ICP, namely 123M ICP (i.e., 46%), is staked for the maximum time of 8 years expressing the long-term commitment of these stakers.
```
Comment (Dfinity-Bjoern) [inline how-it-works/5_governance/00_tokenomics.subpage.md:21]:
```suggestion
As the daily reward amount is independent of the overall amount of staked ICP in the system, participants receive larger rewards if overall staking and participation in governance decreases. This mechanism creates a natural incentive to stake ICP and participate in governance. As of November 2022, 266M ICP is staked, corresponding to 54% of the total supply. A significant part of staked ICP, namely 123M ICP (i.e., 46%), is staked for the maximum time of 8 years expressing the long-term commitment of these stakers.
```

**Review by Dfinity-Bjoern — APPROVED** [review]
That looks great, thanks @bjoernek!

File: how-it-works/5_governance/00_tokenomics.subpage.md (hunk 2 above)
Comment (bjoernek) [reply how-it-works/5_governance/00_tokenomics.subpage.md:27]: @sesi200 : I have uploaded it before via a separate PR, see here https://github.com/dfinity/portal/pull/694/files

**Review by sesi200 — APPROVED** [review]

**Pull request merged**